Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-slim-buster
+FROM debian:10 as build-env
```

Comment:
fwiw this is a switch to python3.7 from 3.8 .. Debian 10 is buster.. I'm assuming distroless, is just using distro deb packages for install.

Reply:
Acknowledged, figured the size reduction of both image and attack surface was worth it the going back to 3.7.  Submitted https://github.com/GoogleContainerTools/distroless/issues/489 requesting Python3.8 image.